.github/workflows/go-test.yaml

```diff
@@ -1,7 +1,10 @@
 name: go-test
 on:
+  pull_request:
   push:
-  workflow_call:
+    branches:
```

Comment:
Instead you can make your release wflow to require test and lint workflows

Reply:
https://github.com/tigrisdata/tigris-client-python/actions/runs/5063675766

Reply:
Updated, check it out.